src/hooks/posters.tsx

```diff
@@ -118,7 +138,7 @@ export const usePosters = (posterHallId: string) => {
       normalizedSearchQuery
     );
 
-    if (tokenisedSearchQuery.length === 0) return filteredPosterVenues;
+    if (!tokenisedSearchQuery.length) return filteredPosterVenues;
```

Comment:
We were explicit in using `===` here on purpose. Please revert.

Reply:
Should have had a comment for the reasoning  above it I guess. What's the purpose?

Reply:
reverted, with a comment above

Reply:
It’s just our style to explicitly check the length rather than check if the length is falsy, no need to comment.

Reply:
Also our style is to post the commit hash where changes were made in the review comments when marking them resolved so that others can easily see where/what was changed. Please do so.

Reply:
> It’s just our style to explicitly check the length rather than check if the length is falsy, no need to comment, so probably remove that again.

Few notes on this statement:

1. There isn't explicit code style (yet, but comments like this are helpful in compiling one) that can read beforehand
2. With no explicit code style, there also isn't explicit reasoning on why strict equality (in type and value) is preferable over falsy value only
3. With no explicit code style comments could exist explaining why something isn't a code smell, like in e.g.
```javascript
    if (!normalizedSearchQuery) return filteredPosterVenues;

    const tokenisedSearchQuery = tokeniseStringWithQuotesBySpaces(
      normalizedSearchQuery
    );

    if (tokenisedSearchQuery.length === 0) return filteredPosterVenues;
```
  Granted, check of a boolean and of a length of an array are different (green to red apples, not oranges), still few comments in code may save further comments in PRs since not everyone at every time will be fully up to speed with the code style and reasoning behind it
4. Current comment was provided just because of prior art, already done one change and reversion w/o readily visible rationale, something in the line of
  // this is how it's supposed to be for the reason of...
  
Let me see if I get this hash thing right: 

Code style update at ecd77d03

Reply:
The solution to that is to make the time to properly document our code style, which we intend to do already. Not to litter the codebase with things that don’t belong there.

(Sarcasm) otherwise we should add a comment to every single line everywhere explaining why we follow that particular code style in that place.

It’s not how it’s done.